conformance_tests/test_tiovx/test_bidir_ext_framework.c

```diff
@@ -157,10 +150,13 @@ vx_status addParameterToGraph(vx_graph graph, vx_node node, vx_uint32 num)
 /* Create a graph with a non-virtual attached to one output, two bidirectional and two inputs */
 void makeGraph1O2B2I(vx_context context)
 {
-    vx_graph graph = vxCreateGraph(context);
-    vx_image image1 = vxCreateImage(context, 100, 100, VX_DF_IMAGE_U8);
-    vx_image image2 = vxCreateImage(context, 100, 100, VX_DF_IMAGE_U8);
-    vx_image image3 = vxCreateImage(context, 100, 100, VX_DF_IMAGE_U8);
+    vx_graph graph;
+    vx_image image1, image2, image3;
+
+    ASSERT_VX_OBJECT(graph = vxCreateGraph(context), VX_TYPE_GRAPH);
```

Comment:
I assume the full conformance test runs through successfully with all of these assertions added?

Reply:
yes and here are the results:
[ ======== ]
[ ALL DONE ] 7252 test(s) from 161 test case(s) ran
[ PASSED   ] 7252 test(s)
[ FAILED   ] 0 test(s)
[ DISABLED ] 9822 test(s)

=================================
OpenVX Conformance report summary
=================================

To be conformant to the OpenVX baseline, 7238 required test(s) must pass. 7238 tests passed, 0 tests failed. PASSED.
To be conformant to the User Data Object extension, 14 required test(s) must pass. 14 tests passed, 0 tests failed. PASSED.
Note: The 9822 disabled tests are optional and are not considered for conformance.